docs/dyndns/update-api.rst

```diff
@@ -83,8 +83,19 @@ determine the hostname, we try the following steps until there is a match:
 - After successful authentication (no matter how), the only hostname that is
   associated with your user account (if not ambiguous).
 
-If we cannot determine a hostname to update, the API will return a ``404 Not
-Found`` status code.
+If we cannot determine a hostname to update, the API will return a ``400 Bad
+Request`` or ``404 Not Found`` status code (depending on the cause).
```

Comment:
400 if the domain could not be uniquely identified and 404 if no matching domain could be found

Reply:
done